File: frontend/types/aircraft.ts
export interface Aircraft {
  icao24: string;
  callsign: string | null;
  origin_country: string | null;
  longitude: number;
  latitude: number;
  barometric_altitude: number | null;
  on_ground: boolean;
  velocity: number | null;
  heading: number | null;
  vertical_rate: number | null;
  last_contact: number;
}

File: backend/Models/Aircraft.cs
namespace AltusIQ.Api.Models;

public class Aircraft
{
    public string Icao24 { get; set; } = string.Empty;
    public string? Callsign { get; set; }
    public string? OriginCountry { get; set; }
    public double? Longitude { get; set; }
    public double? Latitude { get; set; }
    public double? BarometricAltitude { get; set; }
    public bool OnGround { get; set; }
    public double? Velocity { get; set; }
    public double? Heading { get; set; }
    public double? VerticalRate { get; set; }
    public long LastContact { get; set; }
}